pkgs/applications/version-management/gitoxide/default.nix

```diff
@@ -0,0 +1,26 @@
+{ lib, rustPlatform, fetchFromGitHub }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "gitoxide";
+  version = "0.3.0";
+
+  src = fetchFromGitHub {
+    owner = "Byron";
+    repo = "gitoxide";
+    rev = "v${version}";
+    sha256 = "0xpic9jx7nrxi5d8lqch2vxpvipx994d717c4n0kgr3ipyij1347";
+  };
+
+  verifyCargoDeps = true;
```

Comment:
`verifyCargoDeps` was removed from `buildRustPackage`.
```suggestion
```

Reply:
Thanks, I completely missed that development.